### ISSUE TITLE
Default language field in resource templates doesn't validate

### ISSUE THREAD

**Issue by allanaaa** [omeka/omeka-s#1929]
In re: https://github.com/omeka/omeka-s/issues/1886.

As per https://github.com/omeka/omeka-s/issues/1910, you can just put in any old junk for resource templates' default languages.

![RTlanguage](https://user-images.githubusercontent.com/28012148/190271753-e79f33b9-801e-407c-bf9c-dd4806cccf7d.PNG)

This does pass through to items when editing after setting the defaults. (Normally those fields would validate if you manually entered them in the item edit screen.) In the image below, the second description entry saved without any issues and is now displaying proudly as "en, fr" on the public page.

![RTlanguage2](https://user-images.githubusercontent.com/28012148/190271758-770dcbab-8b9b-41ee-8198-1dfbc501c563.PNG)

**Comment by jimsafley** [comment]
This should be fixed once #1919 is merged. You could test this by merging the `validate-language` branch.

**Comment by zerocrates** [comment]
It's now merged into develop.

**Comment by allanaaa** [comment]
Thanks! I saw it go by, thought it had been merged already and this had been missed.
